scripts/validate_docstrings.py

```diff
@@ -240,13 +247,16 @@ def pandas_validate(func_name: str):
             result["errors"].append(
                 pandas_error("EX02", doctest_log=result["examples_errs"])
             )
-        for err in doc.validate_pep8():
+
+        for error_code, error_message, error_count in doc.validate_pep8():
             result["errors"].append(
                 pandas_error(
                     "EX03",
-                    error_code=err.error_code,
-                    error_message=err.message,
-                    times_happening=f" ({err.count} times)" if err.count > 1 else "",
+                    error_code=error_code,
+                    error_message=error_message,
+                    times_happening=f" ({error_count} times)"
+                    if error_count > 1
+                    else "",
```

Comment:
`times_happening` has become pretty long - perhaps define it before the `results['errors'].append(` line, and then here just use the variable you've assigned to